main.go

```diff
@@ -298,6 +298,10 @@ func buildCommand(c *cli.Context) {
 		}
 	}
 
+	if dir, err := os.Stat(contextDir); err != nil || !dir.IsDir() {
+		log.Errorf("Context directory %s is not actualy directory.", contextDir)
```

Comment:
Please, change it to `"Context directory %s is not a directory."`.
Also, you are shadowing `os.Stat` error here. If stat will fail, you will still be showing "not actualy directory" error instead an actual error.